freqtrade/templates/subtemplates/indicators_full.j2

```diff
@@ -2,87 +2,130 @@
+# dataframe["kc_percent"] = (
+#     (dataframe["close"] - dataframe["kc_lowerband"]) /
+#     (dataframe["kc_upperband"] - dataframe["kc_lowerband"])
+# )
+# dataframe["kc_width"] = (
+#     (dataframe["kc_upperband"] - dataframe["kc_lowerband"]) / dataframe["kc_middleband"]
+# )
+
+# # Ultimate Oscillator
+# dataframe['uo'] = ta.ULTOSC(dataframe)
+
+# # Commodity Channel Index: values [Oversold:-100, Overbought:100]
 # dataframe['cci'] = ta.CCI(dataframe)
 
-# MACD
-macd = ta.MACD(dataframe)
-dataframe['macd'] = macd['macd']
-dataframe['macdsignal'] = macd['macdsignal']
-dataframe['macdhist'] = macd['macdhist']
-
-# MFI
-dataframe['mfi'] = ta.MFI(dataframe)
-
-# # Minus Directional Indicator / Movement
-# dataframe['minus_dm'] = ta.MINUS_DM(dataframe)
-# dataframe['minus_di'] = ta.MINUS_DI(dataframe)
-
-# # Plus Directional Indicator / Movement
-# dataframe['plus_dm'] = ta.PLUS_DM(dataframe)
-# dataframe['plus_di'] = ta.PLUS_DI(dataframe)
-# dataframe['minus_di'] = ta.MINUS_DI(dataframe)
-
-# # ROC
-# dataframe['roc'] = ta.ROC(dataframe)
+# RSI
+dataframe['rsi'] = ta.RSI(dataframe)
 
-# # Inverse Fisher transform on RSI, values [-1.0, 1.0] (https://goo.gl/2JGGoy)
+# # Inverse Fisher transform on RSI: values [-1.0, 1.0] (https://goo.gl/2JGGoy)
 # rsi = 0.1 * (dataframe['rsi'] - 50)
 # dataframe['fisher_rsi'] = (np.exp(2 * rsi) - 1) / (np.exp(2 * rsi) + 1)
 
-# # Inverse Fisher transform on RSI normalized, value [0.0, 100.0] (https://goo.gl/2JGGoy)
+# # Inverse Fisher transform on RSI normalized: values [0.0, 100.0] (https://goo.gl/2JGGoy)
 # dataframe['fisher_rsi_norma'] = 50 * (dataframe['fisher_rsi'] + 1)
 
-# # Stoch
+# # Stochastic Slow
 # stoch = ta.STOCH(dataframe)
 # dataframe['slowd'] = stoch['slowd']
 # dataframe['slowk'] = stoch['slowk']
 
-# Stoch fast
+# Stochastic Fast
 stoch_fast = ta.STOCHF(dataframe)
 dataframe['fastd'] = stoch_fast['fastd']
 dataframe['fastk'] = stoch_fast['fastk']
 
-# # Stoch RSI
+# # Stochastic RSI
 # stoch_rsi = ta.STOCHRSI(dataframe)
 # dataframe['fastd_rsi'] = stoch_rsi['fastd']
 # dataframe['fastk_rsi'] = stoch_rsi['fastk']
 
+# MACD
+macd = ta.MACD(dataframe)
+dataframe['macd'] = macd['macd']
+dataframe['macdsignal'] = macd['macdsignal']
+dataframe['macdhist'] = macd['macdhist']
+
+# MFI
+dataframe['mfi'] = ta.MFI(dataframe)
+
+# # ROC
+# dataframe['roc'] = ta.ROC(dataframe)
+
 # Overlap Studies
 # ------------------------------------
 
-# Bollinger bands
+# Bollinger Bands
 bollinger = qtpylib.bollinger_bands(qtpylib.typical_price(dataframe), window=20, stds=2)
 dataframe['bb_lowerband'] = bollinger['lower']
 dataframe['bb_middleband'] = bollinger['mid']
 dataframe['bb_upperband'] = bollinger['upper']
+dataframe["bb_percent"] = (
+    (dataframe["close"] - dataframe["bb_lowerband"]) /
+    (dataframe["bb_upperband"] - dataframe["bb_lowerband"])
+)
+dataframe["bb_width"] = (
+    (dataframe["bb_upperband"] - dataframe["bb_lowerband"]) / dataframe["bb_middleband"]
+)
+
```

Comment:
I would maybe add these in as comment assuming people will do a first try with this - but not use them - so it would be wasted cpu cycles ...

Reply:
I'll take care of sample_strategy.py, thanks.